### ISSUE TITLE
iOS Create New Folder

### ISSUE THREAD

**Issue by wuenschedesign** [owncloud/ios-app#219]
### Steps to reproduce
1. Start ownCloud Application
2. Select an ownCloud account
3. Click on the new-folder-icon

### Expected behaviour
the button »done« should be better called »create«

### Actual behaviour
the button to confirm the action is currently named »Done«

### Client
**iOS version:**
12.0.1

**ownCloud app version:**
1.0 build 85

**Device model:**
iPhone 7

### Screenshots:
Current screen | Possible Solution
-- | --
![bildschirmfoto-2019-01-14-um-18 13 46-a](https://user-images.githubusercontent.com/40062503/51191231-3b6b2b00-18e4-11e9-9498-4f9554add450.png) | ![bildschirmfoto-2019-01-14-um-18 13 46-solution](https://user-images.githubusercontent.com/40062503/51191253-458d2980-18e4-11e9-9e15-ef32175e376f.png)

**Comment by michaelstingl** [comment]
The current behaviour is heavily inspired from the iOS Files app, the central place where users manage files across all apps. Our assumption was, users will learn those patterns.

**Comment by wuenschedesign** [comment]
I updated the screens. As I am not a native speaker I can not tell if »done« is an adequate expression. The iOS App uses it, but I found it confusing, because it is not done yet, it’s done when the folder really exists in the list. The ownCloud App should also adapt the native keyboard. As soon as someone types a name in the text input, the confirm button on the bottom right should switch to signal blue.

**Comment by michaelstingl** [comment]
Good point. “Create” sounds more actionable. This is also what Apple normally recommends. @tomneedham recommendations? @pablocarmu what does Apples HIG say here? Highlighting the bottom-right keyboard button would be also cool. The thing is, you have only a limited number of options. (See @felix-schwarz ‘s 
https://github.com/owncloud/ios-app/issues/48#issuecomment-388752444)

**Comment by michaelstingl** [comment]
BTW I personally don’t like Apple is using the keyboard mode with suggestions in the Files app. We should start to document some decisions…

**Comment by tomneedham** [comment]
"Create" is nicer as it adds information on what will happen afterwards. "Done" is generic.

**Comment by tomneedham** [comment]
Suggestions mode is ok IMO, and there is plenty of space on the screen. Since folders are typically named using full words with correct casing, I think this is ok. Other screens where only lower case, or no space is required, we should not enable this.
